src/pudl/cli.py

```diff
@@ -26,24 +31,21 @@
 logger = logging.getLogger(__name__)
 
 
-def parse_command_line(argv):
-    """
-    Parse script command line arguments. See the -h option.
-
-    Args:
-        argv (list): command line arguments including caller file name.
-
-    Returns:
-        dict: A dictionary mapping command line arguments to their values.
+def create_arg_parser():
+    """Create an argument parser for the etl script."""
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        parents=[pudl.etl.command_line_flags()])
```

Comment:
Jan split command args into etl.py and cli.py. Should all of them live in cli.py?

Reply:
Hm. Is there any practical reason for having the chunk of args in `etl.py`? On the surface it seems much cleaner to pull all of the command args over here.